src/PersonLink.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { EnumPersonType } from './types/EnumPersonType';
+import { Person } from './types/Person';
+
+interface PersonLinkProps {
+  person: Person;
+  handlePersonClick: (name: string | null) => void;
+  personType: EnumPersonType;
+  isPresentInTable?: boolean;
+}
+
+export const PersonLink: React.FC<PersonLinkProps> = (
+  {
+    person, handlePersonClick, personType, isPresentInTable = true,
+  },
+) => {
+  const handleClick = () => {
+    handlePersonClick(person[personType]);
+  };
+
+  return (
+    <>
+      {!isPresentInTable
+        ? person[personType] || '-'
+        : (
+          <Link
+            to={`#/people/${person.slug}`}
+            onClick={handleClick}
+            className={classNames('', {
```

Comment:
The 'classNames' function is used with an empty string as its first argument, which doesn't make sense. If you want to conditionally apply the 'has-text-danger' class, you can just pass the object as the only argument to 'classNames'.